src/v2/view-builder/views/TerminalView.js

```diff
@@ -134,7 +134,7 @@ const Body = BaseForm.extend({
     } else if (this.options.appState.containsMessageStartingWithI18nKey(END_USER_REMEDIATION_KEY_PREFIX)) {
       this.add(createCallout({
         type: 'error',
-        content: new EndUserRemediationTerminalMessage({messages: messagesObjs}),
+        content: new EndUserRemediationMessages({messages: messagesObjs.value}),
```

Comment:
@shawyu-okta Here's the change for the end user remediation error flow to pass in the messages `value` instead